pkgs/servers/keycloak/default.nix

```
@@ -0,0 +1,29 @@
+{ stdenv, fetchzip, makeWrapper, jre }:
+
+stdenv.mkDerivation rec {
+  name    = "keycloak-v${version}";
+  version = "9.0.0";
+
+  src = fetchzip {
+    url    = "https://downloads.jboss.org/keycloak/${version}/keycloak-${version}.zip";
+    sha256 = "1w2d76v1rjghvdks1w32qi08gh88cd37vbf6vx0kq9a2gnhn7hip";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir $out
+    cp -r * $out
+    chmod +x $out/bin/standalone.sh
+    wrapProgram $out/bin/standalone.sh \
+      --prefix PATH ":" ${jre}/bin ;
+  '';
+
+  meta = with stdenv.lib; {
+    homepage    = "https://www.keycloak.org/";
+    description = "Open Source Identity and Access Management For Modern Applications and Services";
+    license     = licenses.asl20;
+    maintainers = [ stdenv.lib.maintainers.ngerstle ];
```

Should be sufficient with
```suggestion
    maintainers = [ maintainers.ngerstle ];
```

---

Fixed, thank you.